### PR TITLE
Feature/mes 3839 Display vehicle checks on debrief 

### PR DESCRIPTION
**Pull request by edwardwilson**
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by faggc — APPROVED** [review]

File: src/pages/debrief/cat-be/components/vehicle-checks-card/__tests__/vehicle-checks-card.spec.ts [hunk 1]
```diff
@@ -0,0 +1,182 @@
+// import { ComponentFixture, async, TestBed } from '@angular/core/testing';
```
Comment (rossfauldsbjss) [inline src/pages/debrief/cat-be/components/vehicle-checks-card/__tests__/vehicle-checks-card.spec.ts]: Tests need to be uncommented and made to work.

**Review by rossfauldsbjss — CHANGES_REQUESTED** [review]
Tests commented out for Cat BE - they are just a copy of Cat B tests and should instead should be Cat BE specific tests

**Comment by ammarhaiderbjss** [comment]
Missing a description as well as a screenshot for this :(

**Comment by rstepnowski** [comment]
You should've rebased that branch against developing - so many changes already merged got pulled from develop on this one + there are still conflicts on this

**Pull request closed without merging**